### PR TITLE
[content update] 742-aws-s3-buckets

### PR DESCRIPTION
**Pull request by DiSanchz**
This pull request contains the proposed updates for the building block [Use AWS S3 Buckets](https://tilburgsciencehub.com/building-blocks/store-and-document-your-data/store-data/aws-s3-buckets/), as part of the task assigned to Issue #742.

### Main comments on the non-updated version

- The 3rd line of code of the box "upload a file" has to be modified from `filetoupload = open("hello.txt", "w")` to `filetoupload = "hello.txt"`
- Overall BB is good and goes straight to the point. Perhaps it could benefit from some more text to complement & contextualize code
- The second link in the "see also" section is wrong. It does not lead to boto3's documentation.

### Implemented modifications

- Keywords expanded.
- Small fix on the code snippet of the section "upload a file" to make it work.
- Short learning goals paragraph added at the beginning of the text.
- Some minor Grammarly suggestions were implemented.
- Second link in the "see also" section was corrected to: https://boto3.amazonaws.com/v1/documentation/api/latest/index.html

### CODE REVIEW

**Review by thierrylahaije — APPROVED** [review]

File: content/building-blocks/store-and-document-your-data/store-data/aws-s3-buckets.md [hunk 1]
```diff
@@ -10,17 +10,21 @@ aliases:
...
 
 If you are using AWS S3 to store data necessary for your research and want to incorporate downloading and uploading your data files to AWS S3 into a [`make`](/building-blocks/configure-your-computer/automation-and-workflows/make/) script, it is useful to use a code script to interact with your AWS S3 bucket instead of the command line. Keep reading to learn how to do it.
 
 {{% tip %}}
 **Some terminology**
 
 You will often read about **AWS S3 buckets**. What are they?
-Put simply, a bucket is a container for objects (files) stored in Amazon S3. Every object must be contained in a bucket. A single user can have as many buckets as they want. Inside a bucket, you can create as many folders you want.
+Put simply, a bucket is a container for objects (files) stored in Amazon S3. Every object must be contained in a bucket. A single user can have as many buckets as they want. Inside a bucket, you can create as many folders as you want.
```
Comment (srosh2000) [inline content/building-blocks/store-and-document-your-data/store-data/aws-s3-buckets.md:27]: I would not put this under a "tip" box.. might be better to integrate it with the para above introducing AWS buckets?

File: content/building-blocks/store-and-document-your-data/store-data/aws-s3-buckets.md [hunk 2]
```diff
@@ -33,15 +37,20 @@ Learn more about AWS credentials [here](https://docs.aws.amazon.com/general/late
 
 ## Code
```
Comment (srosh2000) [inline content/building-blocks/store-and-document-your-data/store-data/aws-s3-buckets.md:38]: Better subtitle: "Set up a connection with AWS S3 using Python" or better instead of "Code"?

File: content/building-blocks/store-and-document-your-data/store-data/aws-s3-buckets.md [hunk 3]
```diff
@@ -115,7 +136,7 @@ s3.download_file(your_aws_bucket_name, "remote-object-name.txt", "local-file-nam
 ## See also
 
 - Learn more about [AWS S3](https://aws.amazon.com/s3)
-- Learn more about [`boto3`](https://aws.amazon.com/s3), the AWS SDK for Python
+- Learn more about [`boto3`](https://boto3.amazonaws.com/v1/documentation/api/latest/index.html), the AWS SDK for Python
 - Learn more on how to upload files with [`boto3`](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/s3-uploading-files.html)
 - Learn more on how to download files with [`boto3`](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/s3-example-download-file.html)
```
Comment (srosh2000) [inline content/building-blocks/store-and-document-your-data/store-data/aws-s3-buckets.md:142]: Might be good to have a more detailed summary section in bullet points with key steps and functions used?

File: content/building-blocks/store-and-document-your-data/store-data/aws-s3-buckets.md (hunk 1 above)
Comment (DiSanchz) [reply content/building-blocks/store-and-document-your-data/store-data/aws-s3-buckets.md]: Perfect! I've integrated the former tip box's content as part of the text right above.

File: content/building-blocks/store-and-document-your-data/store-data/aws-s3-buckets.md (hunk 2 above)
Comment (DiSanchz) [reply content/building-blocks/store-and-document-your-data/store-data/aws-s3-buckets.md]: Good point, indeed the `## Code` header may sound a bit generic. I have substituted it for "Handle AWS S3 through Python" as I think that "Set up a connection with AWS S3 using Python" may enter in conflict with the sub-header immediately below which refers to the the process of establishing a connection with S3 (Which I have renamed as you proposed).

I feel that after these changes are adapt better to the content! 🎯

File: content/building-blocks/store-and-document-your-data/store-data/aws-s3-buckets.md (hunk 3 above)
Comment (DiSanchz) [reply content/building-blocks/store-and-document-your-data/store-data/aws-s3-buckets.md:137]: I've included a new extended summary section featuring a bullet-point format 👍
